docs/docs/coverage/language/c.md

```diff
@@ -12,12 +12,15 @@ The following table provides an outline of the features Trivy offers.
 
 | Package manager | File           | Transitive dependencies | Dev dependencies | [Dependency graph][dependency-graph] | Position |
 | --------------- | -------------- | :---------------------: | :--------------: | :----------------------------------: | :------: |
-| Conan           | conan.lock[^1] |            ✓            |     Excluded     |                  ✓                   |    ✓     |
+| Conan           | conan.lock[^1] |            ✓ [^2]       |     Excluded     |                  ✓ [^3]              |    ✓     |
```

Comment:
Can we split a line?

```suggestion
| Conan (lockfile v1)          | conan.lock[^1] |            ✓       |     Excluded     |                  ✓ [^3]              |    ✓     |
| Conan (lockfile v2)          | conan.lock[^1] |            ✓ [^2]       |     Excluded     |                  -              |    ✓     |
```

Reply:
Updated and rebased to resolve conflicts